picard/mbjson.py

```diff
@@ -441,20 +443,19 @@ def recording_to_metadata(node, m, track=None):
         m['~recordingtitle'] = m['title']
     if m.length:
         m['~length'] = format_time(m.length)
-    if 'instrumental' in m.getall('~performance_attributes'):
-        m.unset('lyricist')
-        m['language'] = 'zxx'
 
 
-def performance_to_metadata(relation, m):
+def get_relation_attributes(relation):
     if 'attributes' in relation:
-        for attribute in relation['attributes']:
-            m.add_unique("~performance_attributes", attribute)
+        return [a for a in relation['attributes']]
```

Comment:
Could we return a set instead? Or does order matter?

Reply:
Probably yes, but I did not want to change that here. It'll obviously mess with order of these attributes in the tags, but the order is kind of undefined anyway. Maybe we should make it a set, but then sort them alphabetically in the tags? That needs an additional processing after all has been parsed, so better do this in a separate PR.